examples/tuning/cta_svm/config_yamls/params/1st_test_acc_params_tuning_config.yaml

```diff
@@ -0,0 +1,65 @@
+type: preprocessor
```

Comment:
Under the highest configuration of test_acc in step2, the parameter range in step3